api/intents_api.py

```diff
@@ -22,21 +22,46 @@
 from framework import basehandlers
 from framework import cloud_tasks_helpers
 from framework import permissions
-from internals import processes
 from internals import stage_helpers
-from internals.core_enums import INTENT_STAGES_BY_STAGE_TYPE
+from internals.core_enums import (
+  INTENT_DRAFT_TYPES_BY_STAGE_TYPE,
+  FEATURE_TYPE_DEPRECATION_ID,
+  IntentDraftType
+)
 from internals.core_models import FeatureEntry
 from internals.review_models import Gate
-from pages.intentpreview import compute_subject_prefix
 import settings
 
 
+SUBJECT_PREFIXES = {
+  IntentDraftType.PROTOTYPE: 'Intent to Prototype',
+  IntentDraftType.DEVELOPER_TESTING: 'Ready for Developer Testing',
+  IntentDraftType.EXPERIMENT: 'Intent to Experiment',
+  IntentDraftType.EXTEND_EXPERIMENT: 'Intent to Extend Experiment',
```

Comment:
This is a much better location for this function. Good call moving it from `intentpreview.py`.